Makefile

```diff
@@ -1,4 +1,5 @@
 SHELL := /bin/bash
+export TERRAFORM_VERSION = 1.3.9
```

Comment:
Any specific reason to pin this version here? I haven't seen this elsewhere, so I'm inclined to say we shouldn't do this.

Reply:
I couldn't make `make lint` pass without this change, as `build-harness` downloads Terraform 1.0 by default ([related code](https://github.com/cloudposse/build-harness/blob/174a591791256c0db03646148850fed50aa1e784/modules/terraform/Makefile#L3)).

I saw this implemented here: https://github.com/cloudposse/terraform-aws-s3-bucket/blob/d3c353f3170347352684b04a75cb62d4bc91ba6e/Makefile#L2

However, I don't think CI runs `make lint`? As https://github.com/cloudposse/terraform-aws-s3-log-storage doesn't include this line, but it requires Terraform `>= 1.3.0`, running `make lint` in that project also fails.